pkg/operation/botanist/kubeapiserver_test.go

```diff
@@ -1529,6 +1543,28 @@ var _ = Describe("KubeAPIServer", func() {
 				HaveKeyWithValue("data-for", []byte("user-kubeconfig")),
 			))
 		})
+
+		It("should not sync the kubeconfig to garden project namespace when enableAdminKubeconfig is set to false", func() {
```

Comment:
Create the kubeconfig secret in the garden cluster before calling `DeployKubeAPIServer` to improve the test accuracy.

Reply:
This is not yet resolved